### PR TITLE
Fix/capture partner

### LINKED ISSUE #493: Yes/No partner answer not coming through to backend

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
fixes #493 

Updated income list section to register whether applicant lived with partner

added action for partner status

removed address and valuation_id from fields as these are already provided. [NEEDS TESTING]

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [ ] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

**Comment by Br3nda** [comment]
This pull request **introduces 1 alert** when merging 58534c444cebf87ec0f0b01991986e89e8aecd5e into a41921cf6d12f2417acb5dc53c61dba0f5bf430a - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/pancake-frontend/rev/pr-c9581b477bc9f6dda0aa9c76d9f0120802a9c4e8)

**new alerts:**

* 1 for Conflicting HTML element attributes

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/components/FirstTimeApplicant/WizardFormSecondPage.js [hunk 1]
```diff
@@ -59,6 +62,8 @@ class WizardFormSecondPage extends React.Component {
   saveFormData() {
     let fields = this.props.formState.form.wizard.values;
     fields['income'] = this.props.storeValues.totalIncome;
+    fields['lived_with_partner'] = this.state.hasPartner;
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/WizardFormSecondPage.js]: why does the variable change name from "has_a_partner" to "lived_with_partner"

Comment (dlouise64) [reply src/components/FirstTimeApplicant/WizardFormSecondPage.js]: @Br3nda good question, looking at this now.

**Comment by Br3nda** [comment]
When i send the application to staging, i get a 500 http error.
I looked in the staging server logs and the error is:
```
2018-08-12T21:46:06.545436+00:00 app[web.1]: E, [2018-08-13T09:46:06.545377 #4] ERROR -- : [59e26aad-5161-4dcc-a40a-9d749ccaf194] ERROR: RuntimeError: No associated property record
2018-08-12T21:46:06.545470+00:00 app[web.1]: E, [2018-08-13T09:46:06.545431 #4] ERROR -- : [59e26aad-5161-4dcc-a40a-9d749ccaf194] app/models/rebate_form.rb:27:in `calc_rebate_amount!'
```

**Comment by dlouise64** [comment]
@Br3nda I returned the valuation_id, the backend is collecting the correct value. All working and ready to test.

**Pull request merged**